src/rp2_common/pico_audio_i2s/audio_i2s.c

```diff
@@ -54,6 +54,7 @@ const audio_format_t *audio_i2s_setup(const audio_format_t *intended_audio_forma
 
 #if PICO_PIO_USE_GPIO_BASE
     if(config->data_pin >= 32 || config->clock_pin_base + 1 >= 32) {
+        assert(config->data_pin >= 16 && config->clock_pin_base + 1 >= 16);
```

Comment:
I think the second part here needs to be `config->clock_pin_base >= 16` because we want _both_ of `clock_pin_base` and `clock_pin_base + 1` to be `>= 16` ? :wink: 
(Currently, this assert wouldn't trigger if `data_pin` is 32, `clock_pin_base` is 15 and `clock_pin_base + 1` is 16)

Reply:
right, I didn't think closely enough about it.